lib/metadata/index.js

```diff
@@ -41,118 +41,104 @@ function inferDownstreamReview(metadata, content) {
     return null;
 }
 
-module.exports = function getMetadada(number, author, title, content) {
```

Comment:
Funny how this was misspelled but did not cause any issues.

Reply:
Yeah the actual name doesn't matter here; it could be an anonymous function if we wanted. This is a bit of an older style of doing module exports I think (not an expert); I believe people tend to export named functions nowadays. (E.g. `exports.getMetadata = function() { ... }`).